OneCommunityGlobal/HighestGoodNetworkApp
# Zijie Lint Fixes for the Final Phase of the Reports Component

# Description
Lint fix for Reports component, this is phase 14（Last Phase）.

## Files changed
src/components/Reports/PeopleReport/PeopleReport.jsx

eslint test:
![CleanShot 2023-10-09 at 00 20 59@2x](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/debe58d7-2213-4c54-a963-13b4f0742f0c)





# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/Reports.jsx`
- `git restore .`
